server/lib/host-metrics.js

```diff
@@ -258,6 +259,49 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(
+  host,
+  { startDate, endDate, collectiveIds = null, timeUnit } = {},
+) {
+  if (!collectiveIds) {
+    const hostedCollectives = await host.getHostedCollectives();
+    collectiveIds = hostedCollectives.map(c => c.id);
+  }
+
+  const results = await sequelize.query(
+    `SELECT t1."amountInHostCurrency" as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
+FROM "Transactions" as t1
+WHERE t1."HostCollectiveId" = :HostCollectiveId
+AND t1."CollectiveId" IN (:CollectiveIds)
+AND t1."kind" IN ('CONTRIBUTION', 'EXPENSE', 'ADDED_FUNDS', 'BALANCE_TRANSFER')
+AND t1."createdAt" >= :startDate AND t1."createdAt" < :endDate
+AND t1."deletedAt" IS NULL
+GROUP BY t1."amountInHostCurrency", t1."hostCurrency", DATE_TRUNC(:timeUnit, t1."createdAt")
+ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
+    {
+      replacements: {
+        ...computeDates(startDate, endDate),
+        timeUnit,
+        HostCollectiveId: host.id,
+        CollectiveIds: collectiveIds,
+      },
+      type: sequelize.QueryTypes.SELECT,
+    },
+  );
+
+  const balanceAtStartDate = await getTotalMoneyManagedAmount(host, {
+    endDate: startDate,
+    collectiveIds,
+    currency: host.currency,
+  });
+  const timeSeries = await convertCurrencyForTimeSeries(results, host.currency);
+  let sum;
+  return timeSeries.map(point => {
+    sum = (sum || 0) + point.amount + balanceAtStartDate.value;
```

Comment:
It's a bit hard to believe that we end up with this when I gave precise guidance on how to compute this metric. https://github.com/opencollective/opencollective-api/pull/6786#discussion_r751090599

It's plain wrong and doesn't make sense if you read it.

Reply:
Ops. 🤯 Seems I am summing the `balanceAtStartDate` repeatedly which I indeed wrong. I've corrected this. Let me know if that looks better. Sorry about that.